### DISCUSSION TITLE
Won't show any Drivers (Hp EliteDesk 705 G4)

### DISCUSSION THREAD

**Discussion by AbdulMaklad** [DavidS95/Smokeless_UMAF#58] [General]
Won't show any Drivers (Hp EliteDesk 705 G4) , tried many options and nothing works, anyone has an idea how to make it work? or it doesn't support EliteDesk or Hp motherboard at all! I actually don't get it, could someone help?

**Comment by angelokimhui** [comment]
I have same problem, I don't see any options at all. I instead have Elitebook 845 G7. It seems like HP has different offset than the common ones. I just instead want to know how to overcome it. Knowing that this software is closed source. I'm not sure as well what to do.

**Comment by kevinhalgren** [comment]
Same issue, HP EliteDesk 705 G4 Mini with Ryzen 5 2400GE
